### ISSUE TITLE
Registration Captcha

### ISSUE THREAD

**Issue by glitchedVixen** [Trans-NRW/forum-theme#1]
The registration captcha is apparently not accessible to users, who need to use a screen reader.

We have to check, if there alternative ways for captchas, which are accessible.

https://forum.trans.nrw/showthread.php?tid=21
